ci/publish-splinter-crates

```diff
@@ -49,11 +49,18 @@ ENV PATH=$PATH:/root/.cargo/bin
 WORKDIR /project/splinter/libsplinter
 
 CMD cargo login $CARGO_CRED \
+ && echo "Publshing version $REPO_VERSION" \
  && rm -f ../Cargo.lock ./Cargo.lock \
  && cargo clean \
  && cargo test \
  && cargo publish \
- && sleep 10 \
+ && bash -c '\
+    while true; do echo "Waiting for Splinter publishing to complete"; \
+    cargo search splinter | grep "splinter = " | grep -q $REPO_VERSION; \
```

Comment:
Will this timeout at any point?

Reply:
No, but given that releases are a manual activity, publishing is generally supervised so an operator can cancel it if necessary.